integrations/launchdarkly/.port/resources/port-app-config.yaml

```diff
@@ -0,0 +1,58 @@
+createMissingRelatedEntities: true
+deleteDependentEntities: true
+resources:
+  - kind: project
+    selector:
+      query: "true"
+    port:
+      entity:
+        mappings:
+          identifier: .key
+          title: .name
+          blueprint: '"launchDarklyProject"'
+          properties:
+            tags: .tags
+  - kind: flag
+    selector:
+      query: "true"
+    port:
+      entity:
+        mappings:
+          identifier: .key
+          title: .name
+          blueprint: '"launchDarklyFeatureFlag"'
+          properties:
+            kind: .kind
+            description: .description
+            creationDate: .creationDate / 1000 | strftime("%Y-%m-%dT%H:%M:%SZ")
+            includeInSnippet: .includeInSnippet
+            clientSideAvailability: .clientSideAvailability
+            temporary: .temporary
+            tags: .tags
+            maintainer: ._maintainer | .firstName + " " + .lastName
+            maintainerEmail: ._maintainer.email
```

Comment:
lets just set the
`maintainer: ._maintainer.email`

No need for maintainerEmail property, remove it from the blueprint properties as well